overlord/hookstate/ctlcmd/is_connected.go

```diff
@@ -55,6 +68,17 @@ func init() {
 	})
 }
 
+func isConnectedPidCheckAllowed(info *snap.Info, plugOrSlot string) bool {
+	slot := info.Slots[plugOrSlot]
+	if slot != nil {
+		switch slot.Interface {
+		case "pulseaudio", "audio-record", "cups-control":
```

Comment:
we need to list these in the help? otherwise how can people know this limitation? or given we have agreed with security that there's no big worry of information leakage we shouldn't limit this? is still true that this doesn't make sense if there's no  peer/socket involved...

Reply:
The main reason I would consider keeping this check is in case we discover some problem with the interface after merging.  It's much easier to change the interface if it is restricted to these fairly uncommon slot types.

I'd be happy to remove it though if you think it isn't needed.

Reply:
I think it's fine to limit it initially as we learn about its usage, but I think these interfaces do need to be listed explicitly in the relevant part of the help for the command otherwise is just confusing.